go-controller/pkg/libovsdb/util/acl.go

```diff
@@ -210,6 +210,7 @@ func UpdateACLLogging(nbClient libovsdbclient.Client, ACLs []*nbdb.ACL, aclLoggi
 
 // ACL L4 Match Construct Utils
 const (
+	NoneMatch = "None"
```

Comment:
nit: doc

Reply:
Now that we are exporting these, I am after better names. I will make suggestions but I'm ok if you think what you have now is the most appropriate ones.

How about "UnspecifiedL4Protocol" and "UnspecifiedL4Match"

I guess the values will have to stay at what they are since they were already exported to DB as IDs. This detail might honor a comment to ensure nobody changes it, barring it being defined in a dedicated DB ID constant block where this would be more clear.

Reply:
I like your suggestions, let me go with them!

Reply:
done!